cmd/server-manager/plugins/events.lua

```diff
@@ -121,24 +128,56 @@ function weatherAPI(raceConfig, serverOpts, apiKey)
             elseif w == 731 then weather["CMGraphics"] = "sol_22_Dust"; weather["CMWFXType"] = 25
             elseif w == 751 then weather["CMGraphics"] = "sol_23_Sand"; weather["CMWFXType"] = 24
             elseif w == 711 then weather["CMGraphics"] = "sol_24_Smoke"; weather["CMWFXType"] = 22
-            elseif w == 300 then weather["CMGraphics"] = "sol_31_Light Drizzle"; weather["CMWFXType"] = 3
-            elseif w == 301 then weather["CMGraphics"] = "sol_32_Drizzle"; weather["CMWFXType"] = 4
-            elseif w >= 302 and w <= 321 then weather["CMGraphics"] = "sol_33_Heavy Drizzle"; weather["CMWFXType"] = 5
-            elseif w == 500 then weather["CMGraphics"] = "sol_34_Light Rain"; weather["CMWFXType"] = 6
-            elseif w == 501 then weather["CMGraphics"] = "sol_35_Rain"; weather["CMWFXType"] = 7
-            elseif w >= 502 and w <= 531 then weather["CMGraphics"] = "sol_36_Heavy Rain"; weather["CMWFXType"] = 8
-            elseif w == 200 or w == 210 or w == 230 then weather["CMGraphics"] = "sol_41_Light Thunderstorm"; weather["CMWFXType"] = 0
-            elseif w == 201 or w == 211 or w == 231 then weather["CMGraphics"] = "sol_42_Thunderstorm"; weather["CMWFXType"] = 1
+			-- For same looking weather without rain, commment the line and uncomment the one below
```

Comment:
I see what you're going for here, could you separate each option into blocks instead of line by line? Just to make it easier to follow and look a bit neater:

-- With rain effects
    _List of elseifs_

-- Without rain effects
    _List of elseifs_

And then comment out the entire rain effects block, I'd like to keep the defaults as they were.

Reply:
No problem :)